resources/init.groovy.d/global-args-security.groovy

```diff
@@ -26,7 +26,7 @@ def qpsPipelineGitBranch = env['QPS_PIPELINE_GIT_BRANCH']
 
 def qpsPipelineLogLevel = env['QPS_PIPELINE_LOG_LEVEL']
 
-def globalPipelineLib = env['GLOBAL_PIPELINE_LIB']
+// def globalPipelineLib = env['GLOBAL_PIPELINE_LIB']
```

Comment:
remove those commented lines completely. the same for manual_deployment/global-args-security.groovy